src/xoak/tests/test_s2_adapters.py

```diff
@@ -16,6 +16,14 @@ def test_s2point(geo_dataset, geo_indexer, geo_expected):
     xr.testing.assert_equal(ds_sel.load(), geo_expected.load())
 
 
+def test_s2point_via_query(geo_dataset, geo_indexer, geo_expected):
```

Comment:
This unnecessarily adds a lot of tests via the parametrized fixtures, but it's probably fine for now.